### DISCUSSION TITLE
AIR GAP installation help needed!

### DISCUSSION THREAD

**Discussion by yanghyeon408** [milvus-io/milvus#22321] [Q&A and General discussion]
Hello,

I'm having trouble with installing milvus on air-gap enviornment.
First, our situation is a docker-compose is not avilable.
So, what i did is ran 3 containers: milvus-standalone, minio, etcd
Having trouble withe the run milvus on milvus-standalone.
Once I run milvus manually in the container, I can see welcome to milvus but the message says cannot access to the config file milvus.yaml and milvus process is not running. 
Once I move to /milvus/configs there is milvus.yaml file but it can't access to the file!!
Anyone can help me?
I'd be very appreciated!
Thank you in advance.

**Comment by yhmo** [comment]
What is your command to start the milvus-standalone container?
